README.md

```diff
@@ -56,6 +56,11 @@ This package can be installed with `pip` from various sources:
     ```bash
     pip install xyzspaces
     ```
+- Install from the [conda](https://anaconda.org/conda-forge/xyzspaces):
```

Comment:
Maybe better: "Install with conda from the Anaconda [conda-forge channel](https://anaconda.org/conda-forge/xyzspaces):"

Reply:
sure, will update.